mw-webapp/src/pages/authPage/AuthPage.tsx

```diff
@@ -1,24 +1,36 @@
+import {getAuth, getRedirectResult, signInWithRedirect, GoogleAuthProvider} from "firebase/auth";
 import {Link} from "react-router-dom";
-import styles from "src/pages/authPage/AuthPage.module.scss";
+import {secondaryApp} from "src/firebase";
 
 export const AuthPage = () => {
+  const provider = new GoogleAuthProvider();
+  const auth = getAuth(secondaryApp);
+
+  const handleGoogleSignIn = async () => {
```

Comment:
Let's make our components as simple as possible, could we move auth logic to the separate hook? Move it please)

Reply:
Done